packages/code/src/Code/Code.styles.ts

```diff
@@ -112,6 +154,47 @@ export function getCodeWrapperVariantStyle(theme: Theme): string {
   `;
 }
 
+export const expandButtonStyle = css`
```

Comment:
Can we add transitions to this button?

Reply:
Is there a transition that we typically use for buttons? I'll have a look and see if I can find something, but if there's a standard here that you could share, that would be helpful.

Reply:
Was there an issue with adding transitions?

Reply:
Oops, nope! I think I just accidentally removed it when I removed the transition for the icon rotation. Will add it back now.